### ISSUE TITLE
Error creating new project

### ISSUE THREAD

**Issue by hakonfjukstad** [ngx-rocket/generator-ngx-rocket#194]
I tried creating a new project 
Got some warnings during the install:
npm install
npm WARN @angular/flex-layout@2.0.0-beta.10-4905443 requires a peer of @angular/core@~4.4.4 but none is installed. You must install peer dependencies yourself.
npm WARN @angular/flex-layout@2.0.0-beta.10-4905443 requires a peer of @angular/common@~4.4.4 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-html@0.11.0 requires a peer of postcss-sass@>=0.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

I try to run and I get a blank page with console output:
[App] init
logger.service.ts:108 [I18nService] Language set to en-US
core.js:3565 Angular is running in the development mode. Call enableProdMode() to enable the production mode.
logger.service.ts:108 [AuthenticationGuard] Not authenticated, redirecting...
core.js:1350 ERROR Error: Uncaught (in promise): EmptyError: no elements in sequence
EmptyError: no elements in sequence
    at new EmptyError (EmptyError.js:28)
    at FirstSubscriber._complete (first.js:154)
    at FirstSubscriber.Subscriber.complete (Subscriber.js:122)
    at MergeMapSubscriber._complete (mergeMap.js:150)
    at MergeMapSubscriber.Subscriber.complete (Subscriber.js:122)
    at MapSubscriber.Subscriber._complete (Subscriber.js:140)
    at MapSubscriber.Subscriber.complete (Subscriber.js:122)
    at ArrayObservable._subscribe (ArrayObservable.js:124)
    at ArrayObservable.Observable._trySubscribe (Observable.js:172)
    at ArrayObservable.Observable.subscribe (Observable.js:160)
    at new EmptyError (EmptyError.js:28)
    at FirstSubscriber._complete (first.js:154)
    at FirstSubscriber.Subscriber.complete (Subscriber.js:122)
    at MergeMapSubscriber._complete (mergeMap.js:150)
    at MergeMapSubscriber.Subscriber.complete (Subscriber.js:122)
    at MapSubscriber.Subscriber._complete (Subscriber.js:140)
    at MapSubscriber.Subscriber.complete (Subscriber.js:122)
    at ArrayObservable._subscribe (ArrayObservable.js:124)
    at ArrayObservable.Observable._trySubscribe (Observable.js:172)
    at ArrayObservable.Observable.subscribe (Observable.js:160)
    at resolvePromise (zone.js:824)
    at resolvePromise (zone.js:795)
    at eval (zone.js:873)
    at ZoneDelegate.invokeTask (zone.js:425)
    at Object.onInvokeTask (core.js:4621)
    at ZoneDelegate.invokeTask (zone.js:424)
    at Zone.runTask (zone.js:192)
    at drainMicroTaskQueue (zone.js:602)
    at <anonymous>

**Comment by sinedied** [comment]
Could you fill in at least environment details from the template?
```
## Environment
- generator version: X.Y.Z
- node version: X.Y.Z  <!-- run `node --version` -->
- npm version: X.Y.Z  <!-- run `npm --version` -->
- OS:  <!-- Mac, Linux, Windows -->
```

The warning during npm install are normal at this point, it won't affect the build.

**Comment by hakonfjukstad** [comment]
generator: 1.6.0 (the one that appears in logo after I do ngx new)
node version: 8.9.1
npm version: 5.5.1
OS: windows 10, 64bit

**Comment by daldegam** [comment]
I have the same bug
```
## Environment
- generator version:  "generator-ngx-rocket"  "version": "3.0.0" [.yo-rc.json]
- node version: v6.10.3
- npm version: 3.10.10
- OS:   Windows 10 64 bits
```

**Comment by sinedied** [comment]
Seems like it's due to the latest `rxjs@5.5.3` update, reverting to `rxjs@5.5.2` works.
```
npm i rxjs@5.5.2
```
And edit your package.json to pin the version until it's resolved:
```
 "rxjs": "5.5.2",
```

See https://github.com/angular/angular/issues/20752 for the source of the issue.

I'll pin the version in the generator tomorrow if it's not fixed meanwhile, thanks for the report.

**Comment by daldegam** [comment]
With version 5.5.2 work! Thank you.

**Comment by sinedied** [comment]
I just released `3.0.1` that pins the rxjs dependency to the working version, as there's still no fixed version out yet.
